collections/src/main/scala/strawman/collection/immutable/LazyList.scala

```diff
@@ -261,10 +225,52 @@ sealed abstract class LazyList[+A]
     if (this.isEmpty) z
     else tail.foldLeft(op(z, head))(op)
   }
+}
+
+trait LazyListOps[+A, +CC[+X] <: LinearSeq[X] with LazyListOps[X, CC, CC[X]], +C <: CC[A]] extends LinearSeqOps[A, CC, C] {
```

Comment:
What do you think of making this trait package private so that we won’t have to support binary compatibility for it?

Reply:
Still needs to be binary-compatible. Types that implement methods leak out.

Reply:
You’re right it’s not really about binary compatibility, but more about not exposing a type that we don’t want people to extend so that we can remove it if it happens to be unnecessary (after we remove `Stream`).